packages/components/src/__future__/Tooltip/ToggleTip/ToggleTip.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { HTMLAttributes, ReactNode, useContext, useRef } from "react"
+import classnames from "classnames"
+import { AriaButtonOptions, useTooltipTrigger } from "react-aria"
+import { TooltipTriggerStateContext } from "react-aria-components"
+import styles from "./ToggleTip.module.scss"
+
+export type ToggleTipProps = {
+  children: ReactNode
+  className?: HTMLAttributes<HTMLButtonElement>["className"]
+} & AriaButtonOptions<"button">
+
+export const ToggleTip = ({
```

Comment:
We should make sure the usage/design guidelines in docs highlight the specific use case this was built for ie. just an icon+tooltip -- to set the expectation that it doesn't support all `Button` cases